src/Take.Blip.Client/Content/InputExpirationTimeDocument.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Lime.Protocol;
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Text;
+
+namespace Take.Blip.Client.Content
+{
+    [DataContract]
```

Comment:
Add some docs here. For instance, what is the purpose of this class/document? Why does it need a property Identity?